app/controllers/facility_accounts_controller.rb

```diff
@@ -1,35 +1,12 @@
 class FacilityAccountsController < ApplicationController
-  module Overridable
-    extend ActiveSupport::Concern
 
-    module ClassMethods
-      def billing_access_checked_actions
-        [ :accounts_receivable, :show_statement ]
-      end
-    end
-
-    def account_class_params
-      params[:account] || params[:nufs_account]
-    end
-
-    def configure_new_account(account)
-      # set temporary expiration to be updated later
-      account.valid? # populate virtual charstring attributes required by set_expires_at
-      account.errors.clear
-
-      # be verbose with failures. Too many tasks (#29563, #31873) need it
-      begin
-        account.set_expires_at!
-        account.errors.add(:base, I18n.t('controllers.facility_accounts.create.expires_at_missing')) unless account.expires_at
-      rescue AccountNumberFormatError => e
-        account.expires_at = Time.zone.now # Prevent expires_at missing message
-      rescue ValidatorError => e
-        account.errors.add(:base, e.message)
-      end
-    end
```

Comment:
I attempted to remove `configure_new_account` in favor of using the new `app/services/account_builder.rb`.  But I'm not sure I successfully supported all use cases.  Please review extra carefully.  Thanks!